Tmain/tag-relative-option.d/run.sh

```diff
@@ -34,8 +34,12 @@ fi
     ${CTAGS} $O --_xformat="yes:     $F -> %F"  --tag-relative=yes    $F
     ${CTAGS} $O --_xformat="always:  $F -> %F"  --tag-relative=always $F
 } | {
-	# Normalize driver letter
-    sed -e 's|C:|/c|g'
+    # Normalize driver letter
+    if command -v cygpath > /dev/null && command -v awk > /dev/null ; then
+        awk '{if ($NF ~ /^[A-Z]:/) { "cygpath "$NF | getline newpath; sub($NF,newpath) } print}'
+    else
+        sed 's|\([A-Z]\):|/\l\1|'
```

Comment:
I think that `\l` is a GNU extension.
I'm not sure we can use it here.

Reply:
Yes, `\l` is a GNU extension. But only Windows's path match `[A-Z]:` pattern, `Cygwin` and `MSYS2` use `GNU sed`

Reply:
You are correct but I feel the technique is a bit tricky.
So could you write ' \l is a GNU extension. But only Windows's path match [A-Z]: pattern, Cygwin and MSYS2 use GNU sed' as a comment at the line?

Reply:
Yes, I can. In addition, If you worried about this may cause problem in some cases, I can use `python` or `perl` to implements `sed 's|\([A-Z]\):|/\l\1|'`